service/lib/agama/storage/config_conversions/to_json_conversions/search.rb

```diff
@@ -33,13 +33,22 @@ def self.config_type
             Configs::Search
           end
 
+          # @see Base#convert
+          def convert
+            converted = super
+            return SEARCH_ANYTHING_STRING if converted.is_a?(Hash) && anything?(converted)
```

Comment:
Is there any case in which `super` does not return a hash?

Reply:
Not really. Good point.